graylog2-web-interface/src/views/components/export/ExportModal.test.tsx

```diff
@@ -204,28 +203,65 @@ describe('ExportModal', () => {
     );
   });
 
+  it('initial fields should be  in the same direction as in widgetConfig where are more than 8 fields', async () => {
+    const widgetConfig = new MessagesWidgetConfig(
+      [
+        'timestamp', 'source', 'gl2_processing_timestamp', 'streams', 'gl2_accounted_message_size', 'controller', 'ingest_time', 'gl2_receive_timestamp', 'user_id',
```

Comment:
Let's create a const for this array inside this test and reuse the fields in line 235.